File: jramirez9.java
//Juan Ramirez-gomez
/*
 * csc321
 * lab#9
 */
import java.util.Random;
import java.util.Scanner;
public class jramirez9
{
	public static void main(String[]args)
	{
		Scanner keyboard = new Scanner(System.in);
		Random random = new Random();
		int rand = random.nextInt(20)+1;
		int num;
	

		do
		{
			System.out.print("Enter a number between 1 - 20: ");
			 num = keyboard.nextInt();

			if(num < 1 || num > 20)
			{
				System.out.print("out of range please try in the range of 1 - 20\n");
			}

		}while(num < 1 || num > 20);

		if(num == rand)
		{
			for(int i = 0; i < num; i++)
			{
				System.out.print("you guess right: \n " + num);
			}
		}
		if(num > rand)
		{
			System.out.print("You guess wrong and its higher than my number " + rand);
		}
		if(num < rand)
		{
			System.out.print("You guess wrong and its lower than my number " + rand);
		}

	}

}

File: jramirez9.c
//Juan Ramirez

#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main(void)
{
	srand(time(NULL));
	int num;
	int rando =(rand() %20) + 1;

	

	
	do
	{
		printf("Enter a number between 1 - 20: ");
		scanf("%d", &num);


		if(num < 1 || num > 20)
		{
			printf("out of range please try in the range of 1 - 20\n");
	
		}


	
		
	}while(num < 1 || num > 20);



		if(num == rando)
                {
			for(int i = 0; i < num;i++)
			{
                        	printf("you guess right: %d\n ", num);
			}
                }
                
               
		 if(num > rando)
                 {
                     printf("you guess wrong and its higher than my number %d", rando);
                     
                 }
		if(num < rando)
                 {
                           printf("you guess wrong and its lower than my number %d", rando);
                           
                  }
                
	

	
	


	return EXIT_SUCCESS;
}
